ceph.spec.in

```diff
@@ -61,6 +78,11 @@ Requires: systemd
 %if 0%{with cephfs_java}
 BuildRequires: sharutils
 %endif
+%if 0%{with selinux}
+BuildRequires:	checkpolicy
+BuildRequires:	selinux-policy-devel
+BuildRequires:	/usr/share/selinux/devel/policyhelp
```

Comment:
The force rebuild won't help, here. The issue is that the policy packages in centos 6.5 do not have a certain macro defined. We can probably disable the (default) SELinux support in centos 6 for now. If there is a demand for SELinux on rhel 6 then we should work on getting the appropriate SELinux bits, there.